js/web/lib/wasm/jsep/webgpu/ops/binary-op.ts

```diff
@@ -188,19 +199,24 @@ export const mul = (context: ComputeContext): void => {
 };
 
 export const pow = (context: ComputeContext): void => {
+  const type = tensorDataTypeToWgslType(context.inputs[0].dataType);
```

Comment:
can use IndicesHelper instead of adding this function:

```
const input = inputVariable('input', context.inputs[0].dataType, context.inputs[0].dims);
const type = input.type.value;
```